raiden/tests/unit/api/test_python_api.py

```diff
@@ -120,3 +120,39 @@ def test_token_swap(raiden_network, deposit, settle_timeout):
         channel(app0, app1, taker_token), deposit + taker_amount, [],
         channel(app1, app0, taker_token), deposit - taker_amount, [],
     )
+
+
+@pytest.mark.parametrize('blockchain_type', ['geth'])
+@pytest.mark.parametrize('channels_per_node', [1])
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_api_channel_events(raiden_chain):
+    app0, app1 = raiden_chain  # pylint: disable=unbalanced-tuple-unpacking
+    token_address = app0.raiden.chain.default_registry.token_addresses()[0]
+    channel_0_1 = channel(app0, app1, token_address)
+
+    amount = 30
+    direct_transfer(
+        app0,
+        app1,
+        token_address,
+        amount
+    )
+
+    results = RaidenAPI(app0.raiden).get_channel_events(channel_0_1.channel_address, 0)
+    assert len(results) == 3
+    max_block = 0
+    for idx, result in enumerate(results):
+        if result['block_number'] > max_block:
+            max_block = result['block_number']
+
+        if idx == 2:
```

Comment:
This test is relying on the number/order of events to never change, is that intended?